api_spec.yaml

```diff
@@ -115,6 +115,7 @@ paths:
       operationId: revokeOAuth2AuthorizationCode
       description: |
         Revoke all authorization codes belonging to a session or to a client (across all tenants).
```

Comment:
This can be converted to a delete API where the query params are sessionHandle or clientId (not both)